filebeat/docs/configuration.asciidoc

```diff
@@ -150,48 +151,59 @@ The buffer size every harvester uses when fetching the file. The default is 1638
 
 ===== max_bytes
 
-Maximum number of bytes a single log event can have. All bytes after max_bytes are discarded and not sent.
-This is especially useful for multiline log messages which can get large. The default is 10MB (10485760).
+The maximum number of bytes that a single log event can have. All bytes after `max_bytes` are discarded and not sent.
+This setting is especially useful for multiline log messages, which can get large. The default is 10MB (10485760).
 
 ===== multiline
 
-Mutiline can be used for log messages spanning multiple lines. This is common for Java Stack Traces
-or C-Line Continuation. The following example combines all lines following a line start with a `[`.
-This could for example be a Java Stack Trace.
+Options that control how Filebeat deals with log messages that span multiple lines. Multiline messages are common in Java stack traces and C line continuation. 
+
+The following example shows how to configure Filebeat to handle a multiline message where the first line of the message begins with a bracket (`[`). 
```

Comment:
filter should work, but expecting people to copy'n paste this, they will not be happy. Normally log files are a mix of arbitrary logs mixed with java (or some other languages) stack traces.

For example this (not checked this works or '\s' is supported), should be more reliable:

    multiline:
        pattern: \s+
        match: after